Sources/NIOHTTP1/HTTPTypes.swift

```diff
@@ -38,13 +38,17 @@ public struct HTTPRequestHead: Equatable {
     /// The header fields for this HTTP request.
     public var headers: HTTPHeaders
 
+    /// The keepAlive that may be parsed by http_parser before. This is only safe to be used as long as
+    /// the user did not modify the headers yet.
```

Comment:
Let's update this comment slightly:

```
/// The keepAlive value that was detected by http_parser. This can only be used safely
/// as long as the user has not made modifications to this header block.
```